applicationinsights.json

```diff
@@ -0,0 +1,11 @@
+{
+  "role": {
+    "name": "dps-data-compliance"
+  },
+  "customDimensions": {
+    "service.version": "${BUILD_NUMBER}"
+  },
+  "selfDiagnostics": {
+    "destination": "console"
```

Comment:
missing the `DEBUG` logging unless you don't want it any more?

Reply:
added